src/app/Home/styles/home.scss

```diff
@@ -0,0 +1,7 @@
+.home__title {
+  color: '#FD5523'
```

Comment:
Can we have all our colors in a separate `constants.scss` file and then import them for use where necessary. We need to pair on this so that we don't have to redo everything when the codebase is too large.